src/modal/modal.ts

```diff
@@ -87,6 +89,9 @@ export class CaptureEyeModal extends LitElement {
         this.modalElement.classList.remove('modal-visible');
         closeButton.classList.add('close-button-hidden');
         closeButton.classList.remove('close-button-visible');
+        if (hasTouchScreen()) {
+          closeButton.classList.add('mobile');
+        }
```

Comment:
I'm not sure if it would be better to move the mobile logic to the 'else' part (not modalHidden), since the mobile icon is visible when the modal is not hidden.